### ISSUE TITLE
compile error prevents mongo initial setup

### ISSUE THREAD

**Issue by arnoudb** [Ekliptor/WolfBot#29]
hi, very interesting bot you created. Specs and flexibility are promising. Unfortunately i can't get it to run locally to test it out. 

You mention that there are some expected compile errors but this one seems to fall in to another category. when running `tsc` this one is the one preventing me to run the bot:

```
tests/init/init.ts:42:36 - error TS2339: Property 'getInitFunctions' does not exist on type 'typeof import("/Users/arnoudbos/Documents/crypto/projects/bots/WolfBot/node_modules/@ekliptor/bit-models/build/models/Liquidation")'.

42     addInitFn(actions, Liquidation.getInitFunctions(db.get()));
```

command to start the bot (hoped it may not be a showstopper):
 
`node app.js --config=Back../config/HoneyBadger.json --trader=RealTimeTrader --noUpdate --noBrowser`

But then then the output is:
2019-09-09 00:29:02 - info: Connected to MongoDB
2019-09-09 00:29:02 - info: minified all resources
2019-09-09 00:29:02 - info: Initializing database: Creating collections on first start
2019-09-09 00:29:02 - warn: Unhandled Rejection
2019-09-09 00:29:02 - warn:  {"message":"bit_models_1.Liquidation.getInitFunctions is not a function","stack":"TypeError: bit_models_1.Liquidation.getInitFunctions is not a function\n    at Object.<anonymous> (/Users/arnoudbos/Documents/crypto/projects/bots/WolfBot/tests/init/init.ts:42:36)\n    at Generator.next (<anonymous>)\n    at fulfilled (/Users/arnoudbos/Documents/crypto/projects/bots/WolfBot/build/tests/init/init.js:8:58)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)"}
2019-09-09 00:29:02 - warn: TypeError: bit_models_1.Liquidation.getInitFunctions is not a function
    at Object.<anonymous> (/Users/arnoudbos/Documents/crypto/projects/bots/WolfBot/tests/init/init.ts:42:36)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/arnoudbos/Documents/crypto/projects/bots/WolfBot/build/tests/init/init.js:8:58)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)

So basically can't setu the mongo db for first time usage.

Regards, Arnoud

**Comment by Ekliptor** [comment]
Hi
that was an old version of `bit-models` - i just updated the latest to NPM.
Please run `yarn update` and ensure you have `bit-models 1.0.67` installed.
Afterwards run `tsc` (compile Typescript) in the bit-models folder.
